### ISSUE TITLE
<svg> without <title> child element doesn't violate accessibility standards

### ISSUE THREAD

**Issue by vbence86** [metal/metal-a11y-checker#12]
## Disclamer
https://www.w3.org/TR/SVG/access.html defines that `<svg>` element must have a `<title>` child that explains the purpose of the SVG.

## Steps
Execute the validator against the following HTML snippet
```html
<!DOCTYPE html>
<html lang="en">
<head>
	<title>Bence</title>
</head>
<body>
	<svg>
		<use xlink:href="{$spritemap}#{$symbol}" />
	</svg>
</body>
</html>
```

## Expected
Accessibility violation is found

## Actual
No accessibility violation is found.
